APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
Ah.  Because most of the time, when I use it, I already have a debugPrinter object allocated, so I've found it to be less code, and more readable, to simply ask the printer object to format something for me.